scripts/data_catalog.py

```
@@ -0,0 +1,161 @@
+    
+    parts = file_path.stem.split('_')
+    if len(parts) >= 3:
+        info['country'] = parts[0].upper()
+    
+    if 'weather' in file_path.name or 'airquality' in file_path.name:
+        try:
+            if '_to_' in file_path.name:
+                date_parts = file_path.stem.split('_')
+                for i, part in enumerate(date_parts):
+                    if part == 'to' and i > 0 and i < len(date_parts) - 1:
+                        info['start_date'] = datetime.strptime(date_parts[i-1], '%Y%m%d').strftime('%Y-%m-%d')
```

Ruff (0.12.2)

Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)